Sources/Fluent/SQL/GeneralSQLSerializer.swift

```diff
@@ -363,6 +363,8 @@ open class GeneralSQLSerializer: SQLSerializer {
             return "BOOL"
         case .data:
             return "BLOB"
+        case .datetime:
```

Comment:
`timestamp` might be more accurate. `time` is just the time without date or timezone.